_targets.R

```diff
@@ -76,16 +79,85 @@ min_obs_days <- 100
 # Change dummy date to force re-build of NWIS DO sites and data download
 dummy_date <- "2022-06-15"
 
-# test and validation sites
+
+#2) Configure model inputs/variables 
+
+# Define test and validation sites
 val_sites <- c("01472104", "01473500", "01481500")
 tst_sites <- c("01475530", "01475548")
 
 train_start_date <- '1980-01-01'
-train_end_date <- '2017-01-01'
-val_start_date <- '2017-01-01'
-val_end_date <- '2019-01-01'
-test_start_date <- '2019-01-01'
-test_end_date <- '2022-01-01'
+train_end_date <- '2014-10-01'
+val_start_date <- '2014-10-01'
+val_end_date <- '2015-10-01'
+test_start_date <- '2015-10-01'
+test_end_date <- '2022-10-01'
+
+# Define model parameters and combine within a list that gets used to
+# write a base model config file for the snakemake modeling workflow.
+base_config_options <- list(
+  out_dir = "../../../out/models",
+  # random seed for training; If FALSE, no seed. Otherwise, specify the seed:
+  seed = FALSE,
+  num_replicates = 1,
+  trn_offset = 1,
+  tst_val_offset = 1,
+  epochs = 100,
+  hidden_size = 10,
+  dropout = 0.2,
+  recurrent_dropout = 0.2,
+  finetune_learning_rate = 0.01,
+  early_stopping = FALSE,
+  # train/val/test split information is defined above:
+  val_sites = val_sites, 
```

Comment:
```suggestion
  validation_sites = val_sites, 
```
 `validation` was what was in the config files originally

Reply:
when I ran this, it threw an error, b/c Snakemake was looking for the `validation_sites` config option.

Reply:
Great catch, thanks! I've accepted your suggested fix in https://github.com/USGS-R/drb-do-ml/pull/163/commits/a9da5c3f43be883eab57709947c4c5cc9d3aa976.